Tests/js/views/services/assets/ez-dashboardblocksviewservice-tests.js

```diff
@@ -0,0 +1,226 @@
+            });
+            this.wait();
+        },
+
+        'Should fail when loading root location model fails': function () {
+            var rootLocationId = 'root-location';
+
+            Y.Mock.expect(this.discoveryService, {
+                method: 'getInfoObject',
+                args: ['rootLocation', Y.Mock.Value.Function],
+                run: function (identifier, callback) {
+                    callback(false, {
+                        _href: rootLocationId
+                    });
+                }
+            });
+
+            Y.Mock.expect(this.rootLocationModel, {
+                method: 'set',
+                args: ['id', rootLocationId],
+                returns: this.rootLocationModel
+            });
+
+            Y.Mock.expect(this.rootLocationModel, {
+                method: 'load',
+                args: [Y.Mock.Value.Object, Y.Mock.Value.Function],
+                run: Y.bind(function (options, callback) {
+                    Y.Assert.areSame(this.capi, options.api, 'Should pass correct API to the `load` method of root location model');
+
+                    callback(true);
+                }, this)
+            });
+
+            this.service.on('error', Y.bind(function (event) {
+                this.resume(function () {
+                    Y.Assert.areSame(loadLocationModelErrorMessage, event.message, 'Should show correct error message');
+                });
+            }, this));
+            this.service.load(function () {
+                Y.Assert.fail('The service should not be loaded');
+            });
+            this.wait();
+        },
+    });
+
+    getViewParametersTest = new Y.Test.Case({
+        name: 'eZ Dashboard Blocks View Service get view parameters test',
+
+        setUp: function () {
+            this.rootLocationModel = new Y.Mock();
+
+            this.service = new Y.eZ.DashboardBlocksViewService({
+                rootLocation: this.rootLocationModel
+            });
+        },
+
+        tearDown: function () {
+            this.service.destroy();
+        },
+
+        'Should return a correct set of data when running `getViewParameters` method': function () {
+            Y.Assert.areSame(this.rootLocationModel, this.service.getViewParameters().rootLocation, 'Should return `rootLocation` model');
+        }
+    });
+
+    rootLocationDefaultValueTest = new Y.Test.Case({
+        name: 'eZ Dashboard Blocks View Service root location default value test',
+
+        setUp: function () {
+            this.rootLocationModel = new Y.Mock();
+
+            this.service = new Y.eZ.DashboardBlocksViewService();
+        },
+
+        tearDown: function () {
+            this.service.destroy();
+        },
+
+        'Should return a model': function () {
+            Y.Assert.isInstanceOf(Y.Model, this.service.get('rootLocation'), 'Should return a model');
+        }
+    });
+
+    searchPluginRegisterTest = new Y.Test.Case(Y.eZ.Test.PluginRegisterTest);
```

Comment:
this is a wrong place to do that, since you are testing some code added in the search plugin module, the corresponding test should go the search plugin module test so here: https://github.com/ezsystems/PlatformUIBundle/blob/master/Tests/js/views/services/plugins/assets/ez-searchplugin-tests.js#L779 (where while at it you can add the others view services on which the search plugin is plugged)